domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131700__expenditure_seed_data

```diff
@@ -1,12 +1,12 @@
-db.eAT.insertMany([
+db.expenditure.insertMany([
 {
     "_id" : "910f4d23-fc63-48fb-a8fb-f0ad567b788a",
-    "_class" : "org.egov.web.models.EAT",
-    "tenantId" : "HR",
+    "_class" : "org.egov.web.models.Expenditure",
```

Comment:
Do we need class attribute? If not mandatory, please remove.

Reply:
It takes automatically, in order to remove it we have do conversion in all entities. that will take time for whole refactoring thing.

Reply:
Ok. Let it be like this for now.